setup.py

```diff
@@ -28,9 +29,9 @@
 setuptools.setup(
     name="Rare",
     version=version,
-    author="Dummerle",
+    author="RareDevs",
     license="GPL-3",
-    description="A gui for Legendary",
+    description="A gui for legendary",
```

Comment:
What's up with lowercasing these strings? They were correct as they were

Reply:
The capitalization was inconsistent between files. `gui` should also be `GUI` technically but looks ugly as a window title. I'd like to find some other descriptive text for Rare anyways, as both `GUI` and `Legendary` are not descriptive enough for unfamiliar users.